po/de.po

```diff
@@ -1423,15 +1510,16 @@ msgstr "Passwort:"
 msgid "Filename:"
 msgstr "Dateiname:"
 
+# TODO: Check if it looks fine
 #: GTG/gtk/backends/parameters_ui/period.py:48
 msgid "Check for new tasks every"
-msgstr "Überprüfe auf neue Aufgaben jede"
+msgstr "Neue Aufgaben jede"
```

Comment:
Maybe "alle" instead of "jede" would sound better.
Also "Auf neue" instead of "Neue" might be good.